resources/lib/play.py

```diff
@@ -41,8 +40,12 @@ def play(url):
         # set in program.url, otherwise we fetch it separately
         if p.get_url():
             stream_url = p.get_url()
+            sub_url = None
         else:
-            stream_url = comm.get_stream(p.id)
+            stream_info = comm.get_stream(p.id)
+            stream_url = stream_info['url']
+            sub_url = stream_info['subtitles']
+            print sub_url
```

Comment:
Use utils.log() if you want it printed. Also, you should look always use print() instead